scripts/elior.coffee

```diff
@@ -18,15 +19,22 @@ module.exports = (robot) ->
         else
           data = JSON.parse body
           if day < 0
-            chat.reply (data.map (dayMenu) -> formatMenu dayMenu).join '\n'
+            cb (data.map (dayMenu) -> formatMenu dayMenu).join '\n'
           else if day < 5
-            chat.reply formatMenu data[day]
+            cb formatMenu data[day]
 
   robot.respond /menu$/, (chat) ->
-    getMenu chat, 0
+    getMenu 0, (msg) -> chat.reply msg
   robot.respond /menu du jour$/, (chat) ->
-    getMenu chat, 0
+    getMenu 0, (msg) -> chat.reply msg
   robot.respond /menu de demain$/, (chat) ->
-    getMenu chat, 1
+    getMenu 1, (msg) -> chat.reply msg
   robot.respond /menu de la semaine$/, (chat) ->
-    getMenu chat, -1
+    getMenu -1, (msg) -> chat.reply msg
+
+  # each work day @ 11:50, the day's menu
+  new HubotCron '50 11 * * 1-5', 'Europe/Paris', () ->
+    getMenu 0, (msg) -> robot.messageRoom 'qbot-dev', msg
```

Comment:
Each monday it will send a message at 10 and at 11h50, which is floody

IMO a cron just for the current day is enough, no need to print the one for the week. Otherwise, keep the week on monday but do not print again at 11h50